modal/image.py

```diff
@@ -375,7 +375,7 @@ def _assert_no_mount_layers(self):
                 "\n"
                 "Run `image.add_local_*` commands last in your image build to avoid rebuilding images with every local "
                 "file change. Modal will then add these files to containers on startup instead, saving build time.\n"
-                "If you need to run other build steps after adding local files, set `copy=True` to copy the files"
+                "If you need to run other build steps after adding local files, set `copy=True` to copy the files "
                 "directly into the image, at the expense of some added build time.\n"
```

Comment:
Nice thanks for catching this. I've adopted habit of putting spaces at the beginning of the next line when doing implicit string concatenation because they'll line up and it's easier to see when you're missing one, e.g. 

```
                "If you need to run other build steps after adding local files, set `copy=True` to copy the files"
                " directly into the image, at the expense of some added build time.\n"
```

We're definitely not consistent across the codebase and I don't think you need to revisit this, just FYI in case it feels like a better approach going forward.